services/web/client/source/class/osparc/utils/Utils.js

```diff
@@ -128,12 +128,7 @@ qx.Class.define("osparc.utils.Utils", {
       return data;
     },
 
-    computeServiceRetrieveUrl: function(srvUrl) {
-      const urlRetrieve = srvUrl + "/retrieve";
-      return urlRetrieve.replace("//retrieve", "/retrieve");
-    },
-
-    computeServiceV2RetrieveUrl: function(studyId, nodeId) {
+    computeServiceRetrieveUrl: function(studyId, nodeId) {
       const urlBase = window.location.protocol + "//" + window.location.host + "/v0";
```

Comment:
I'm not 100% sure this works for every legacy dynamic services we support. Please, check with the usual suspects: 3D viewer, 2D plot

Reply:
So 2D plot seems ok with the new API.
**3D viewer** is not happy, more investigation is required. Apparently it returns a 404 Not found when queried by the backend (log appears in the webserver).